app/src/main/java/nl/babbq/conference2015/AboutActivity.java

```diff
@@ -86,7 +86,8 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 startActivity(i1);
                 break;
             case 2: // Authors
-                // TODO make this a list view of both authors and links to both our Twitters
+                Intent i2 = new Intent(getApplicationContext(), AuthorsActivity.class);
+                startActivity(i2);
```

Comment:
I'm not a fan of `i2` as a variable name, but it matches the pattern in this class, so :neutral_face: